contracts/UnionZap.sol

```diff
@@ -35,9 +30,18 @@ contract UnionZap is Ownable, UnionBase {
         0xCdF7028ceAB81fA0C6971208e83fa7872994beE5;
     address private constant T_ETH_POOL =
         0x752eBeb79963cf0732E9c0fec72a49FD1DEfAEAC;
+    address private constant UNISWAP_ROUTER =
+        0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D;
+    address private constant UNIV3_ROUTER =
+        0xE592427A0AEce92De3Edee1F18E0157C05861564;
+    address private constant WETH_TOKEN =
+        0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+
+    address[] public outputTokens;
 
     uint256 private constant BASE_TX_GAS = 21000;
```

Comment:
`BASE_TX_GAS` and `FINAL_TRANSFER_GAS` can be removed since they aren't used

Reply:
Fixed in 927d350423444a39dba0e73631b8a6cf6d9284c9